pymatgen/io/lobster/inputs.py

```diff
@@ -54,25 +54,24 @@
 
 
 class Lobsterin(UserDict, MSONable):
-    """
-    This class can handle and generate lobsterin files
-    Furthermore, it can also modify INCAR files for lobster, generate KPOINT files for fatband calculations in Lobster,
+    """Handle and generate lobsterin files.
+    Furthermore, it can also modify INCAR files for LOBSTER, generate KPOINTS files for fatband calculations in LOBSTER,
     and generate the standard primitive cells in a POSCAR file that are needed for the fatband calculations.
     There are also several standard lobsterin files that can be easily generated.
-    """
 
-    # reminder: lobster is not case sensitive
+    Reminder: lobsterin keywords are not case sensitive.
+    """
 
-    # keyword + one float can be used in file
-    FLOAT_KEYWORDS = (
+    # Keyword + float
+    _FLOAT_KEYWORDS: tuple[str, ...] = (
         "COHPstartEnergy",
         "COHPendEnergy",
         "gaussianSmearingWidth",
         "useDecimalPlaces",
         "COHPSteps",
     )
-    # one of these keywords +endstring can be used in file
-    STRING_KEYWORDS = (
+    # One of these keywords + string
```

Comment:
Maybe something like this?

```suggestion
    # These keywords need an additional string suffix
```

Reply:
Sure!